updated the profile's image of management

isaachuangx opened the pull request:
adding the image made by DALLE for management profile.

isaachuangx committed:
updated the profile's image of management

adding the image made by DALLE for management profile.

GaneshSrambikal commented:
This pr can be closed

isaachuangx committed:
created the Contact.html and updated code with all
info

isaachuangx committed:
Update Contact.html

using the card style from bootstrap and divide content into 3 parts. upper part is the image. adding the border or lines to separating those part. The Bottom to left is image part. The right is text content.

isaachuangx committed:
Update Contact.html

update the text contents

isaachuangx committed:
Update Contact.html

adjusted the card size
modified card inner down side area which the left image area become larger the right side of context part comes a little bit smaller.

isaachuangx committed:
update the admin and login process

add the HOST and CSRF in setting.
try to add some logging response status in views of app for login process. and add logout function

GaneshSrambikal commented:
> adding the image made by DALLE for management profile.

Hello, 
You can merge your PRs/changes into your main branch of the forked repo. As the original repo changes can only be merged by the author
unless you are collaborating.
Thank you.

isaachuangx committed:
logout function

try to use the POST method instead of GET from the lab. clean the route problem of urls.py in both the djangoapp and djangoproj. Also, had changed some params in views.py of djangoapp

isaachuangx committed:
add registration function

adding the Resgiter.jsx. configured routes of app.js, urls.py in 'proj' . and creats the def registration(request) in app/views.py

isaachuangx committed:
Update urls.py

correct the path in app/urls.py for register instead of registration

isaachuangx committed:
adding new endpoints

completed all the endpoint for getting dealership data.

isaachuangx committed:
updated the fetchReviews/dealer/:id

According to  the dealership.json ,findById(id) since the key of "id"was not generated by MongoDB. So it will switch to findOne({ id: id })

isaachuangx committed:
created models

created models in model.py and registered in admin.py

isaachuangx committed:
up views,urls,populate

 add a method to get the list of cars，add the path for get_cars，populated when the first call is made to get_cars

isaachuangx committed:
back end interaction

fixed the error in models. created communication with backend

isaachuangx committed:
updating .env,urls,views

replaced the url in env
updated the path in urls
getting new fetching method in views

isaachuangx committed:
updated the views and restapis

updateing methods of receiving dealership's reviews

isaachuangx committed:
add a path

isaachuangx committed:
updated path

updated path of url and route in app.js

isaachuangx committed:
updated the logout function

fixed the logout problem

isaachuangx committed:
correcting some errors

isaachuangx committed:
Update .env

isaachuangx committed:
Update .DS_Store

isaachuangx committed:
updated urls

isaachuangx committed:
reconstruct the Dealer.jsx

there was a render problem in Dealer.jsx

isaachuangx committed:
modified app.js and path

try to solve problem of initializing a database. Since created a new DB, it is invalid to fetch any  data

isaachuangx committed:
logger

adding logger into views.py

isaachuangx committed:
updated the app.js

modified the fetch method

isaachuangx committed:
Update .env

isaachuangx committed:
Update app.js

isaachuangx committed:
Update .env

isaachuangx committed:
adding loggers

the loggers would helps identifies de issue of url

isaachuangx committed:
Update urls.py

isaachuangx committed:
add logs in frontend

try to see the url output send out correctly

isaachuangx committed:
Update urls.py

isaachuangx committed:
Update Dealers.jsx

try to simplify the url definitions

isaachuangx committed:
try to fixing the car model fetching issue

there r some issues in the post review page while the car model data can not be fetched

isaachuangx committed:
Revert "try to fixing the car model fetching issue"

This reverts commit 92c1a301a136313907f4abf6adaff99e35f7adff.

isaachuangx committed:
Revert "Update Dealers.jsx"

This reverts commit bba6d95d8e00cdbd5a4c9815208f9ded7882ca16.

isaachuangx committed:
Update .env

isaachuangx committed:
add login and register button

adding more loggers and try to add login and register button in the dealership listing page.

isaachuangx committed:
Revert "add login and register button"

This reverts commit ce09779490e4a62c591492d22fdd56a822beb806.

isaachuangx committed:
adding login and register button

adding the link in all page currently

isaachuangx committed:
adding links

isaachuangx committed:
Update .env

isaachuangx committed:
Update admin.py

isaachuangx committed:
update for lint check

isaachuangx committed:
Update review.js

isaachuangx committed:
Update review.js

isaachuangx committed:
Update review.js

isaachuangx committed:
overcoming the es6 check

isaachuangx committed:
solving es6 problem

isaachuangx committed:
Revert "solving es6 problem"

This reverts commit 55c900847cd7229e3fe8898dc1e177432d2cf905.

isaachuangx committed:
Revert "overcoming the es6 check"

This reverts commit f683cf28818a1a5c35d727a0c9b0e7859364caea.

isaachuangx committed:
adding es6 config

isaachuangx committed:
created eslint.config.js

isaachuangx committed:
fix es lint server

isaachuangx committed:
update lint

isaachuangx committed:
Update review.js

isaachuangx committed:
Revert "update lint"

This reverts commit 2d868898c06014995926cb964f914b6acf0dd25c.

isaachuangx committed:
reverted couple changes

isaachuangx committed:
deleted eslint config

isaachuangx committed:
Revert "deleted eslint config"

This reverts commit ae336884ff4a2c23e3ee916419df98340d7d3479.

isaachuangx committed:
created .jshintrc

isaachuangx committed:
Update .jshintrc

isaachuangx committed:
lint test

isaachuangx committed:
lint test 1

isaachuangx committed:
lint test 2

isaachuangx committed:
lint test 3

isaachuangx committed:
linter test 4

isaachuangx committed:
linter test 5

isaachuangx committed:
simple test 1

isaachuangx committed:
Update App.js

isaachuangx committed:
Revert "Update App.js"

This reverts commit 4c920f961537bfb0df4a9c352b65222c04f4eb62.

isaachuangx committed:
Revert "simple test 1"

This reverts commit a7da17671790ca6983501e1897d109d0f472a388.

isaachuangx committed:
Revert "linter test 5"

This reverts commit 2016153976de80c242def78310240669aa90c116.

isaachuangx committed:
Revert "linter test 4"

This reverts commit 99cef387582b97bbb3f2b8bf57fdba6559417244.

isaachuangx committed:
Revert "lint test 3"

This reverts commit 5dd3f4bb38375a04cb6992f522fd7e37630aabd7.

isaachuangx committed:
Revert "lint test 2"

This reverts commit 98758b34020d2c5c2ff73127529b674574354f1e.

isaachuangx committed:
Revert "lint test 1"

This reverts commit 3dfc6478594fa12904b591b69669a23a3ce522a0.

isaachuangx committed:
Revert "lint test"

This reverts commit baad13fd6acf4eb7985c128f1293fd5e4ba43f29.

isaachuangx committed:
Create main.yml

isaachuangx committed:
Remove unused Int32 variable to fix lint error

isaachuangx committed:
Merge branch 'D1' of https://github.com/isaachuangx/xrwvm-fullstack_developer_capstone into D1

isaachuangx committed:
Fix lint errors in models.py

isaachuangx committed:
Fix lint errors in populate.py

isaachuangx committed:
Fix lint errors in restapis.py

isaachuangx committed:
Fix lint errors in urls.py

isaachuangx committed:
Fix lint errors in views.py

isaachuangx committed:
Fix lint errors in urls.py

isaachuangx committed:
Fix lint errors in views.py

isaachuangx committed:
Fix lint errors in settings.py

isaachuangx committed:
Fix trailing whitespace errors in urls.py

isaachuangx committed:
Fix line length errors in urls.py

isaachuangx committed:
Fix trailing whitespace and line length errors in urls.py

isaachuangx committed:
Fix line length errors in urls.py

isaachuangx committed:
fixed render issue

It won't rendered properly when user clicks home page from any other page except page of about and contact. we replace the static page to React method.

isaachuangx committed:
Update App.js

isaachuangx committed:
Update .env

isaachuangx committed:
change a nav-bar colors

there is a nav-bar color issue which refreshing home, about and contact pages with stick with the previous color.

isaachuangx committed:
Update .env

isaachuangx committed:
updated the header color issue

imports the header background color from two css file which make it easy to manage

isaachuangx committed:
creating car inventory api

isaachuangx committed:
car inventory DB

deploy the inventory data base and completed the API endpoint in base on the car_record.json

isaachuangx committed:
updated car_record

the original json file missed price, which i had download the right one

isaachuangx committed:
building the server side data request

packed the api from app.js into the get_inventory()in views.py, then connect the path in urls.py. optimize all the code.

isaachuangx committed:
Update .env

isaachuangx committed:
adding the search bar

a search bar by state for filtering the dealership

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
Update .env

isaachuangx committed:
search car inventory

add the car inventory viewing function on frontend. And solved the two database was not able to running at the same time.

isaachuangx committed:
updated the docker-compose

isaachuangx committed:
fixing the url of fetching car

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
Update .env

isaachuangx committed:
adding dropdown menu

for filtering the car inventory

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
more functions added

adding couple function:
“back” link within dealer details page and car inventory page.
"reset" button for reset the filters.
fix the issue of 2 filter factor matching problem.

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
Update SearchCars.jsx

isaachuangx committed:
Update SearchCars.jsx
